testutil/integration/suites/interface.go

```diff
@@ -0,0 +1,45 @@
+//go:build integration
+
+package suites
+
+import (
+	"testing"
+
+	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+
+	"github.com/pokt-network/poktroll/testutil/integration"
+)
+
+// IntegrationSuite is an interface intended to be used within test suites which
+// exercise an integration.App.
+type IntegrationSuite interface {
+	// NewApp constructs a new integration app and sets it on the suite.
+	NewApp(*testing.T, ...integration.IntegrationAppOptionFn) *integration.App
+	// SetApp sets the integration app on the suite.
+	SetApp(*integration.App)
+	// GetApp returns the integration app from the suite.
+	GetApp() *integration.App
+	// GetPoktrollModuleNames returns the list of the names of all poktroll modules
```

Comment:
I understand why you have `poktroll` in the name now, but would like to avoid it. Let's call it `All` and `Cosmos` is explicit

Reply:
Can you elaborate on your rationale? I feel strongly about avoiding ambiguity around "all".

Reply:
Fair enough. We'll change if/when we need to.